test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -227,6 +233,7 @@ def _test_large_scale_exporter(
         create_model: Callable,
         create_args: Callable,
         create_pytorch_only_kwargs: Callable,
+        dynamic_axes: bool = True,
```

Comment:
```suggestion
        enable_dynamic_shape: bool = True,
```

Reply:
I would actually vote for dynamic_axes. I think this way of naming is more conventional in Python where we don’t say “enable”, “use”, “is” etc. Aka options don’t quite start with verbs.

Reply:
As an argument, it can sometimes be Union[bool, Dict[int, str]] for example. It will just be a matter of not choosing a conflicting name in the function body. 

For example, we can name the argument dynamic_shape so there’s no conflict

Reply:
Now dynamic shape can be less accurate than dynamic axes because dynamic shape also implies the rank is dynamic?

Reply:
It's ok to use `enable_dynamic_axes`. However, as mentioned above, code with dynamic shape behavior can happen 100 times more than the use of this flag. Which one do you think should take the better name? Imagining that if I author 1000 custom ops (not rare because aten has more ops) for dynamic shape, do you want the 1000 custom ops to use `local_dynamic_axes` or just `dynamic_axies` in theyr core logics?

Reply:
So I checked the old API, and we did use verb with bool keyword arguments, I will align with that format. However, I will put a NOTE for us to discuss this again when the real exporter API PR is out.

Reply:
ChatGPT likes `enable_dynamic_axes`.

By ChatGPT:

Both options can be used to name a boolean variable that turns on/off a feature in Python. However, enable_dynamic_axes is generally a more commonly used naming convention in Python.

The use of the word "enable" in the variable name makes it clear that the variable controls whether or not a feature is enabled. Additionally, by using the prefix "enable_", it follows the convention of prefixing boolean variable names with verbs that indicate the action the variable controls. This is a common naming convention in Python, making code more readable and easier to understand for other developers.

On the other hand, dynamic_axes does not clearly indicate that it is a boolean variable and what it controls. Therefore, it might cause confusion to other developers who are not familiar with the code.

Therefore, I would recommend using enable_dynamic_axes as the variable name.

Reply:
> It's ok to use `enable_dynamic_axes`. However, as mentioned above, code with dynamic shape behavior can happen 100 times more than the use of this flag. Which one do you think should take the better name? Imagining that if I author 1000 custom ops (not rare because aten has more ops) for dynamic shape, do you want the 1000 custom ops to use `local_dynamic_axes` or just `dynamic_axies` in theyr core logics?

I would like to point out two things from this scenario: 1) is that we don’t expect this argument to suddenly appear in 1000 ops, if at all. It is an exporter toggle and the ops are expected to support, on or off. 2) the naming concerns mostly the public interfaces and that’s usually a few levels higher than where the logic actually is. We should still be able to strike a balance and not resort to something like `local_dynamic_axes`

Reply:
I also overlooked that this is an internal function, in which case I don’t have a strong opinion

Reply:
Right this not yet a public API. I am resolving this conversation.